src/components/LeafletMap/style.scss

```diff
@@ -0,0 +1,181 @@
+.map-component {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 100;
+
+  .leaflet-container {
+    min-height: 100vh;
```

Comment:
Should this be either `min-height: 100%` or `min-height: inherit`?

Reply:
Seems to only be working with 100vh. Maybe a leaflet thing?